.travis/build.sh

```diff
@@ -10,6 +10,7 @@ SYSTEMTEST_ARGS=${SYSTEMTEST_ARGS:-SmokeTest}
 
 if [ "$VERSION" != "latest" ]; then
     TAG=$VERSION
+    mvn versions:set -DnewVersion=$VERSION
```

Comment:
Does the change to the .pom files actually get committed (and tagged) in git?

Reply:
No, I'm not sure we want to do that